crawl-ref/source/ng-wanderer.cc

```diff
@@ -704,6 +789,11 @@ static void _maybe_memorise_start_spell(const vector<spell_type> &spells)
             ++lvl_1s;
             to_memorise = s;
         }
+        // give stones for use with sandblast
+        if (s == SPELL_SANDBLAST)
+        {
+            newgame_make_item(OBJ_MISSILES, MI_STONE, 10 + random2avg(41, 5));
```

Comment:
Ooh, fans will be excited! (Also, don't forget to run `util/unbrace` and `util/checkwhite`.)